docs/getting-started/using-ale/isd-generate.md

```diff
@@ -0,0 +1,147 @@
+
+# Generating an ISD with isd_generate
+
+!!! success "Goal"
+
+    Generate Instrument Support Data (ISD) for an image with ALE's `isd_generate.py`
+
+## Prerequisites
+
+You will need three things before you can generate an ISD from an image:
+
+- [Installation of ISIS](../../how-to-guides/environment-setup-and-maintenance/installing-isis-via-anaconda.md)
+- [Installation of ALE](https://github.com/DOI-USGS/ale?tab=readme-ov-file#setting-up-dependencies-with-conda-recommended)
+- An Image in ISIS format with SPICE data:
+    - [Locate an Image](../../getting-started/using-isis-first-steps/locating-and-ingesting-image-data.md)
+    - [Ingest the Image into ISIS](../../getting-started/using-isis-first-steps/locating-and-ingesting-image-data.md#introduction-to-importing)
+    - Add Spice Data to the image with [`spiceinit`](https://isis.astrogeology.usgs.gov/Application/presentation/Tabbed/spiceinit/spiceinit.html)*
+    - Download [SPICE data](https://naif.jpl.nasa.gov/naif/data.html) and set `$ALESPICEROOT`*
```

Comment:
This should be higher on the list.

Reply:
Do you always need NAIF Data/`$ALESPICEROOT`, or just for NaifSpice drivers?  (and for IsisSpice drivers, would you have to use `spiceinit` instead?

Reply:
ISISSPICE does not need kernels, anything with naif spice would. Most people would be using naif spice realistically.